src/components/Tabs.tsx

```diff
@@ -0,0 +1,29 @@
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+interface Props {
+  tabs: Tab[],
+  tabId: string;
+}
+
+export const Tabs:React.FC<Props> = ({ tabs, tabId }) => {
+  // const { tabId = 0 } = useParams();
+  const isSelected = (tab: Tab) => tab.id === tabId;
```

Comment:
would be better to have TabContent and Tabs as 1 component, so that you use one variable selecteTab for both of them

Reply:
not fixed